app/packages/core/src/components/Grid/useRefreshers.ts

```diff
@@ -0,0 +1,59 @@
+import { subscribe } from "@fiftyone/relay";
+import * as fos from "@fiftyone/state";
+import { useEffect, useMemo } from "react";
+import uuid from "react-uuid";
+import { useRecoilValue } from "recoil";
+import { gridPage } from "./recoil";
+
+export default function useRefreshers() {
```

Comment:
A refresh resets either the layout or the page _and_ layout

Reply:
Would love to have these comments in the code itself...doesn't have to be polished and formal.